less/opendev.less

```diff
@@ -12,6 +12,7 @@ img {
 }
 
 table {
```

Comment:
@Huyeng how would you suggest that we render that table? I can only think about enabling datatables (with the responsive extension) while on mobile. otherwise the screen width is just to small to accomodate such a table..

In any case I believe it is better to render it like this rather than allowing the table to get out of the horizontal limits of the page.

any ideas?